readme.txt

```diff
@@ -1,23 +1,26 @@
 === Cookie Wow WordPress ===
 Contributors: Cookie Wow
-Tags: cookiewow
-Requires at least:
+Tags: cookie, PDPA
+Requires at least: 4.9.16
 Tested up to: 5.6.2
```

Comment:
The screenshot below shows the plugin was tested with WordPress 5.6.2.

<img width="800" src="https://user-images.githubusercontent.com/4145121/109456359-3dc07d00-7a8b-11eb-814b-ff826fe0c91f.png">